jax/_src/scipy/eigh.py

```diff
@@ -98,7 +98,7 @@ def _split_spectrum_jittable(P, H, V0, rank, precision):
     H: Matrix to be projected.
     V0: Accumulates the isometries into the projected subspaces.
     rank: Rank of P.
-    precision: The matmul precision.
+    precision: :class:`~jax.lax.Precision` object specifying the TPU matmul precision.
```

Comment:
I might omit "TPU" from these docstrings, since clearly something we *should* do is make this apply on GPU if it does not already.

Reply:
My suggestion would be just to document it on `lax.Precision`.